lib/hllcounter.cc

```diff
@@ -259,7 +259,7 @@ double ep_sum(double acc, int b)
 
 int get_rho(HashIntoType w, int max_width)
 {
-    return max_width - floor(log2(w));
+    return max_width - floor(log2(w.as_ull()));
```

Comment:
@camillescott same as below, is converting down from >64bit to 64bit the right thing to do?